Updated buttons.vue

smolinari opened the pull request.

smolinari committed:
Update PULL_REQUEST_TEMPLATE.md

rstoenescu committed:
Update PULL_REQUEST_TEMPLATE.md

smolinari committed:
Merge branch 'dev' of https://github.com/quasarframework/quasar into dev

smolinari committed:
Updated top stepper example. Corrected icon.

smolinari committed:
Corrected spelling mistake.

smolinari committed:
Updated icon to remove testing junk and updated stepper example 1

smolinari committed:
Some more small improvements for the example

smolinari committed:
Some minor improvements, added tooltip to top left and right toggles.

smolinari committed:
Small stepper changes. Merge branch 'dev' of https://github.com/quasarframework/quasar into dev

smolinari committed:
Added inner-loading example.

smolinari committed:
Updated inner-loading.vue

smolinari committed:
Merge branch 'dev' of https://github.com/quasarframework/quasar into dev

rstoenescu committed:
Update inner-loading.vue

smolinari committed:
Merge branch 'dev' of https://github.com/quasarframework/quasar into dev

smolinari committed:
Merge branch 'dev' of https://github.com/quasarframework/quasar into dev

smolinari committed:
Merge branch 'dev' of https://github.com/quasarframework/quasar into dev

smolinari committed:
Update date to buttons.vue

smolinari committed:
Merge branch 'dev' of https://github.com/quasarframework/quasar into dev

rstoenescu commented:
Hey, I'd prefer keeping the current examples for testing purposes. The real examples will be part of Quasar Play, so that's where we can do something styling on what we show as demo.

smolinari commented:
Ok. Though, are the tests automated and need those exact examples? 

Because, all the examples are basically still the same, I just added one more button to show a customized spinner and I also added a click event to one button in the row of spinners examples, so all of the buttons do spin when clicked on. I also added a caption for the row of spinner buttons to show they are spinners. Nothing more really. 😄 

Scott

smolinari commented:
WTF! Oh my. Naturally not what I wanted. Ok. I'll fix and re-PR.

Scott
